.github/workflows/build_docker_images.yml

```diff
@@ -79,3 +83,12 @@ jobs:
           context: atd-etl/cr3_extract_diagram
           push: true
           tags: atddocker/atd-vz-cr3-extract:${{ github.ref == 'refs/heads/master' && 'production' || 'latest' }}
+
+      - name: Build and push CR3 download image
+        if: steps.changes.outputs.cr3_download == 'true'
+        uses: docker/build-push-action@v4
+        with:
+          platforms: linux/amd64,linux/arm64
+          context: atd-etl/cr3_download
+          push: true
+          tags: atddocker/vz-cr3-download:${{ github.ref == 'refs/heads/master' && 'production' || 'latest' }}
```

Comment:
i don't think this is the place to change this pattern, but my two cents is that it would be more clear/explicit to default to a tag name of `staging` instead of `latest` for these images.

Reply:
If someone opens a PR with changes in these folders, it will build the image tagged with the other tag, so latest/staging. I was thinking we shouldn't overwrite images tagged `staging` with code that hasn't been reviewed or approved, but I don't think we have a staging data ETL setup, so it wouldn't affect anything.